libcontainer/configs/validate/validator.go

```diff
@@ -128,6 +128,44 @@ func (v *ConfigValidator) cgroupnamespace(config *configs.Config) error {
 	return nil
 }
 
+// convertSysctlVariableToDotsSeparator can return sysctl variables in dots separator format.
+// The '/' separator is also accepted in place of a '.'.
+// Convert the sysctl variables to dots separator format for validation.
+// More info:
+//   https://man7.org/linux/man-pages/man8/sysctl.8.html
+//   https://man7.org/linux/man-pages/man5/sysctl.d.5.html
+// For example:
+// Input sysctl variable "net/ipv4/conf/eno2.100.rp_filter"
+// will return the converted value "net.ipv4.conf.eno2/100.rp_filter"
+func convertSysctlVariableToDotsSeparator(val string) string {
+	if val == "" {
+		return val
+	}
+	sepFunc := func(r rune) bool {
+		return r == '.' || r == '/'
+	}
+	firsSepIndex := strings.IndexFunc(val, sepFunc)
```

Comment:
Why not use `strings.IndexAny`?

Reply:
There's also a typo: `firs` -> `first`

Reply:
Yes, this function `strings.IndexAny` is more convenient, I have modified it.

`firs` -> `first`： I didn’t pay attention, it has been modified.